### DISCUSSION TITLE
How to change the value of React-Select using React-Hook-Form

### DISCUSSION THREAD

**Discussion by beverlylaurel** [react-hook-form/react-hook-form#4973] [General] [answered]
I'm using react-hook-form to manage my forms
and also using react-select for my select elements
What I want to do is actually very simple but I just couldn't do it and I'm stuck here.

I want to change the value of react-select with external conditions but how do I do this with react-hook-form

The code sample I've written. I want to get the selected value on the radio and set it to react-select

https://codesandbox.io/s/react-hook-form-controller-v5-forked-ndh54?file=/src/index.jsx

note: if the react-select value changes, the radio does not need to be changed either. the main purpose is to manipulate react-select from radio

**Comment by bluebill1049** [comment, marked as answer]
you can use `setValue` api

https://react-hook-form.com/api/useform/setvalue

**Reply by beverlylaurel** [reply]
it worked. thank you so much!

**Reply by bluebill1049** [reply]
np. 👍
